code/datums/spells/devil.dm

```diff
@@ -131,14 +131,6 @@
 	playsound(get_turf(src), 'sound/misc/enter_blood.ogg', 100, 1, -1)
 	var/obj/effect/dummy/slaughter/s_holder = new(loc)
 	ExtinguishMob()
-	if(buckled)
-		buckled.unbuckle_mob(src,force=1)
-	if(has_buckled_mobs())
-		unbuckle_mob()
-	if(pulledby)
-		pulledby.stop_pulling()
-	if(pulling)
-		stop_pulling()
```

Comment:
The reason a lot of this is removed is that `forceMove` handles unbuckling mobs these days; there's still a few places this snowflake check exists, because things like `do_teleport` are really really old code and use things like `move` without `forceMove` so...there's issues.